scripts/setup-smoke-config.sh

```diff
@@ -6,7 +6,7 @@ set -o errexit
 mkdir -p clients
 cat >> smoke/internal/testdata/admin_settings.yml <<EOF
 credentials:
-  github: "$GITHUB_TOKEN"
+  github: "token $GITHUB_TOKEN"
```

Comment:
We don't have docs for it, but our code expects the token to have this token word in front of it. I didn't change our code since we still rely on the oauth token in deployment for fallback